__tests__/test.js

```diff
@@ -44,42 +44,46 @@ var path_asset_cache_buster = function(http_path, real_path, done) {
     var extname = path.extname(http_path)
       , basename = path.basename(http_path, extname)
       , dirname = path.dirname(http_path)
-    
+
     done({path: path.join(dirname, basename + '-v123') + extname, query: null})
   }, 10)
 }
 
 var files = fs.readdirSync(sassDir)
 
-describe('basic', function() {
-  files.forEach(function(file) {
-    test(file, function(done) {
-      equalsFileAsync(file, 'basic', {}, done)
-    })
-  })
-})
+describe.each(['node-sass', 'sass'])('with require("%s")', function(impl) {
```

Comment:
Just for the record, the test failure was not due to the jest version ( it also failed with `jest@16.0.2` with `['node-sass', 'sass'].forEach` instead of `describe.each` )